scenario/clickactionbutton.go

```diff
@@ -133,23 +133,36 @@ func (settings ClickActionButtonSettings) Execute(sessionState *session.State, a
 		return
 	}
 
+	label = firstNonEmpty(label, "buttonaction")
+
 	// run button-actions
 	for _, buttonAction := range buttonActions {
-		err := buttonAction.execute(sessionState, actionState, connectionSettings, label)
+		buttonActionLabel := fmt.Sprintf("%s: %s", label, firstNonEmpty(buttonAction.ActionLabel, buttonAction.ActionType.String()))
```

Comment:
when using %s the stringer interface will be used automatically, i.e. .String() is unnecessary

Reply:
`buttonAction.ActionType.String()` is an argument to `firstNonEmpty`

Reply:
ah, you're right, sorry